dcos_test_utils/ssh_client.py

```diff
@@ -26,42 +26,48 @@
 
 
 class Tunnelled():
-    def __init__(self, base_cmd: list, target: str):
+    def __init__(self, opt_list: list, target: str, port: int):
         """
         Args:
-            base_cmd: list of strings that will be evaluated by check_call
-                to send commands through the tunnel
+            opt_list: list of SSH options strings. E.G. '-oControlPath=foo'
             target: string in the form user@host
+            port: port number to be used for SSH or SCP
         """
-        self.base_cmd = base_cmd
+        self.opt_list = opt_list
         self.target = target
+        self.port = port
 
     def command(self, cmd: list, **kwargs) -> bytes:
-        """ Run a command at the tunnel target
-        Args:
+        """ Run a command at the tunnel target Args:
```

Comment:
Not sure what this formatting change was for.

```
Run a command at the tunnel target

Args:
    cmd: ...
```

Reply:
Yeah, even I think its weird that I did this....